packages/compass-crud/src/stores/crud-store.ts

```diff
@@ -1345,6 +1343,28 @@ class CrudStoreImpl
       countOptions,
     });
 
+    // Only check if index was used if query filter or sort is not empty
```

Comment:
Does it make sense to run explain if there was a sort but no filter? Wouldn’t that always be a collscan anyway?

Reply:
Yeah, you’re right. It does still fetch the entire collection, but I guess if the sort is combined with e.g. a limit, then that’s valid usage of the index. Makes sense, thank you!